### DISCUSSION TITLE
AI object detection

### DISCUSSION THREAD

**Discussion by yogi251579** [community/community#145190] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Could someone recommend a suitable camera for object detection in an industrial setting? I'm looking for a camera that would provide high accuracy and reliability for industrial applications, capable of handling various environmental conditions and offering good integration options for object detection tasks

**Comment by rajaman18** [comment]
you can use Gopro For that .It's potable and good at clarity and detailing. Best for that.
